README.md

```diff
@@ -10,3 +10,24 @@
 ## Licence
 
 Apache 2
+
+## Releasing
+
+We don't use `zest.releaser` right now because of some problems with
+releasing a package that is not at the root of the repo (`jobs/`), and
+because we have no interest in uploading this project to PyPI, but
+this could change if we figure out how.
+
+The current procedure is:
+
+* Bump version in `jobs/setup.py`
+* Update the release date in `jobs/CHANGELOG.rst`
+* `make build-dependencies`
+* `git commit -am "Bump x.y.z"`
+* Open PR, wait for it to become green
+* Merge PR
+* `git tag x.y.z`
+* `git push --tags origin`
+* `make lambda.zip`
+* Add a release on Github with the lambda.zip attached
```

Comment:
What does this mean? Going to https://github.com/mozilla-services/buildhub/releases/new and manually upload the `lambda.zip` file with the browser?

Reply:
Based on previous releases, I think so, but I've never done it myself.

Reply:
Yes, manual upload so far.